client/src/test/suite/completion.test.ts

```diff
@@ -0,0 +1,43 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+import * as vscode from 'vscode';
+import * as assert from 'assert';
+import { getDocUri, activate } from './helper';
+
+suite('Completion tests', () => {
+	const docUri = getDocUri('empty.drl');
+
+	test('Completes "package" at the beginning', async () => {
+		await testCompletion(docUri, new vscode.Position(0, 0), {
+			items: [
+				{ label: 'package', kind: vscode.CompletionItemKind.Keyword }
+			]
+		});
+	}).timeout(5000); // increase timeout from the default 2000ms because helper.activate waits 2000ms for server startup
```

Comment:
In this PR, I add only one test for simplicity. I'll add more tests in later PRs.